doc/user/content/sql/show-sinks.md

```diff
@@ -16,14 +16,35 @@ aliases:
 
 ## Details
 
+Field | Use
+------|-----
+_schema&lowbar;name_ | The schema to show sinks from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
+**FULL** | Return details about your sinks.
+
 ### Output format
 
-`SHOW SINKS`'s output is a table with one column, `name`.
+`SHOW FULL SINKS`'s output is a table, with this structure:
+
+```nofmt
+ name  | type | volatile
+-------+------+---------
+ ...   | ...  | ...
+```
+
+Field | Meaning
+------|--------
+**name** | The name of the sink
+**type** | Whether the sink was created by the `user` or the `system`
+**volatile** | Does the sink depend on any [volatile sources](/overview/api-components/#volatility)?
```

Comment:
If we're supporting `null`, it's probably worth explicitly noting here, as well as whether we're (initially) assigning the volatility value ourselves or whether the user has to do it.

Same suggestion for other affected `SHOW` commands.

Reply:
Added a bit about `NULL`, but not sure how to phrase the rest in this tiny table. Was hoping the link'd be sufficient!